docs/Cloud_integrations/AWS_IoT_Core/Configuring_AWS_integration.md

```diff
@@ -31,51 +31,38 @@ To start integrating AWS with {{ coiote_short_name }}, you first need to create
-![Add REST user](images/add_rest_user2.png "Add REST user")
-    - Click **Save**.
-    - Go to the **Credentials** tab, type a password for your user (twice), select **Set password**, then confirm by clicking **Set password** in the pop-up.
-
-## Copy tasks and provide credentials for your device group in {{ coiote_short_name }}
-
-The {{ coiote_short_name }}-side configuration of the integration is located in the dedicated `AWSiotCoreCertAuth` device group. To complete this side of the integration, log in as the user with the **awsiottenant** role (only if that user was created in the Root **Domain**. If not, they can not access the root groups and the tasks have to be copied from the **Cloud admin** account.).
-Then follow the steps below:
-
-1. Go to the **Device groups** panel and select a group:
-    - For the default setting, select the **AWSiotCoreCertAuth** group which already contains all the necessary tasks and setting values.
-    - Alternatively, create a new group and migrate the required tasks and setting values:
-        - Select the **Add** button, name your group and click **Add**.
-        ![Add group button](images/add_group_button.png "Add group button")
-        - Migrate all the five tasks that have the **AWS** prefix in their task name:
-            - Select the **AWSiotCoreCertAuth** group and go to **Group tasks**, select the first **AWS** task and click **Copy**.
-              ![Copy task](images/copy_task.png "Copy task")
-            - In the pop-up window, click **Select group** in the **Task target** field and choose your custom integration group from the list.
-            - !!! important
-                  Remember to select the **Domain** of the user you created earlier.
-            - In the **Actions** field, select **Add new task**.
-              ![Copy task pop-up](images/copy_task_popup.png "Copy task pop-up")
-            - Repeat the action for the remaining four tasks.
-        - Migrate the `AWSdataPlaneEndpointAddress` setting value:
-            - Select your custom integration group and go to **Profiles**, then select **Copy from**.
-              ![Copy setting values](images/copy_svs.png "Copy setting values")
-            - In the pop-up window, click **Select group** and select the **AWSiotCoreCertAuth** group.
-            - Pick the `AWSdataPlaneEndpointAddress` setting value from the list by checking it in the **Selected** column, then click **Copy**.
-              ![Copy setting values pop-up](images/copy_sv_popup.png "Copy setting values pop-up")
-2. Enter your AWS Endpoint Name in {{ coiote_short_name }}:
-    - Go to **Device groups**, select your custom integration group (or the **AWSiotCoreCertAuth** group, depending on the previous step) and go to **Profiles**. Complete the **AWS** setting value:
+    - Remember to switch off the **Require e-mail verification after first login** and switch on 
+      **Activate user account after creation** toggle buttons.
```

Comment:
There is also need for that used has no "terms and condition acceptance" pending action, becuase when is it enabled on user it has no access to api v3